src/browser/modules/Stream/Auth/ConnectionForm.jsx

```diff
@@ -315,6 +315,9 @@ export class ConnectionForm extends Component {
           username={this.state.username}
           password={this.state.password}
           database={this.state.requestedUseDb}
+          allowedAuthMethods={
+            this.props.inCloudEnv ? [NATIVE] : [NATIVE, NO_AUTH]
+          }
```

Comment:
To be consistent, would you consider moving this to `mapStateToProps`, next to `allowedSchemes`?

Reply:
Sure!